ccmlib/scylla_node.py

```diff
@@ -194,7 +196,10 @@ def _start_jmx(self, data):
         env_copy = os.environ
         env_copy['SCYLLA_HOME'] = self.get_path()
 
+        message = f"Starting scylla-jmx: args={args}"
+        self.debug(message)
```

Comment:
I think debug print in one place should be enough, wouldn't it ?

Reply:
For scylla we have a debug log that goes to the test log and scylla itself echoes its args to its system log.

I find it useful to have it in both places...

Reply:
o.k.